Tools/cases_generator/generate_cases.py

```diff
@@ -199,6 +235,7 @@ class Instruction:
     block: parser.Block
     block_text: list[str]  # Block.text, less curlies, less PREDICT() calls
     predictions: list[str]  # Prediction targets (instruction names)
+    block_line: int  # First line of block in original code
```

Comment:
Why do we need this? Isn't all the location information present in the token stream of the parsed instruction?